shared/login/register/paper-key/index.render.desktop.js

```diff
@@ -1,44 +1,49 @@
-import React, {Component} from 'react'
-import {TextField, RaisedButton} from 'material-ui'
+import React from 'react'
+import {Text, Icon, Input, Button} from '../../../common-adapters'
+import Container from '../../forms/container'
 
-export default class PaperKeyRender extends Component {
-  render () {
-    return (
-      <div style={styles.container}>
-        <h1>Register with a paper key</h1>
-        <h2>Lorem ipsumLorem ipsumLorem ipsumLorem ipsumLorem ipsumLorem ipsumLorem ipsumLorem ipsum Lorem ipsum </h2>
-        <TextField
-          style={{width: '100%'}}
-          hintText='Enter your paper key'
-          floatingLabelText='Paper Key'
-          onEnterKeyDown={() => this.props.onSubmit()}
-          ref='paperKey'
-          onChange={event => this.props.onChangePaperKey(event.target.value)}
-          value={this.props.paperKey}
-        />
-        <RaisedButton
-          style={{alignSelf: 'flex-end', marginTop: 20}}
-          label='Submit & Log in'
-          primary
-          onClick={() => this.props.onSubmit()}
-          enabled={this.props.paperKey}
-        />
-      </div>
-    )
-  }
-}
-
-PaperKeyRender.propTypes = {
-  onSubmit: React.PropTypes.func.isRequired,
-  onChangePaperKey: React.PropTypes.func.isRequired,
-  paperKey: React.PropTypes.string
+const Render = ({onBack, onSubmit, onChangePaperKey, paperKey}) => {
+  return (
+    <Container
+      style={styles.container}
+      onBack={() => onBack()}>
+      <Text type='Header' style={styles.header}>Type in your paper key:</Text>
+      <Icon type='fa-file-text-o' style={styles.icon}/>
+      <Input
+        multiLine
+        style={styles.input}
+        floatingLabelText='opp blezzard tofi pando agg whi pany yaga jocket daubt bruwnstane hubit yas'
```

Comment:
i can but then it got weird, seeing doubled up stuff and how it worked just was weird. the ideal interaction would be you see the fake paperkey if you're not focused. you focus and you get the floating that says 'Paper key' or something